src/tests/Microsoft.PowerFx.Interpreter.Tests/DisplayNameTests.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Text;
+using Microsoft.PowerFx.Core.Tests;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Interpreter.Tests
+{
+    public class DisplayNameTests
+    {
+        [Fact]
+        public void DisplayNameTest1()
+        {
+            var pfxConfig = new PowerFxConfig(Features.SupportIdentifiers);
+            var engine = new RecalcEngine(pfxConfig);
+
+            var recordType = RecordType.Empty()
+                .Add(new NamedFormulaType("logicalA", FormulaType.Number, displayName: "displayName"))
+                .Add(new NamedFormulaType("logicalB", FormulaType.Number));
+
+            var rv1 = RecordValue.NewRecordFromFields(
+                new NamedValue("logicalA", FormulaValue.New(1)),
+                new NamedValue("logicalB", FormulaValue.New(4)));
+            var rv2 = RecordValue.NewRecordFromFields(
+                new NamedValue("logicalA", FormulaValue.New(2)),
+                new NamedValue("logicalB", FormulaValue.New(5)));
+            var rv3 = RecordValue.NewRecordFromFields(
+                new NamedValue("logicalA", FormulaValue.New(3)),
+                new NamedValue("logicalB", FormulaValue.New(6)));
+
+            var tv = TableValue.NewTable(recordType, rv1, rv2, rv3);
+
+            var parameters = RecordValue.NewRecordFromFields(
+                new NamedValue("myTable", tv));
+
+            var result = engine.Eval("DropColumns(myTable, displayName)", parameters);
+           
+            var sb = new StringBuilder();
+            result.ToExpression(sb, new FormulaValueSerializerSettings() { UseCompactRepresentation = true });
+            var output = sb.ToString();
```

Comment:
>sb.ToString(); [](http://example.com/codeflow?start=25&length=14)

nit: there's an overload to simplify:
output = result.ToExpression();

Reply:
It's not fully equivalent as is usesUseCompactRepresentation=false (and the API is missing a bool param)
That said, it doesn't change the test and i'll fix it.